server.py

```diff
@@ -56,12 +62,23 @@ async def delete_url(alias: str):
     if(sqlite_helpers.delete_url(DATABASE_FILE, alias)):
         return {"message": "URL deleted successfully"}
     else:
+        logging.error("url not found")
```

Comment:
i believe the uvicorn logs will show when we return 404, so we can make this debug
```suggestion
        logging.debug(f'url "{alias}" was not found')
```

Reply:
done